e2e-tests/ukef/cypress/e2e/journeys/portal/amendments/change-cover-end-date-journey.happy-path.spec.js

```diff
@@ -64,8 +64,7 @@ context('Change cover end date journey - happy path', () => {
   });
 
   beforeEach(() => {
-    cy.clearSessionCookies();
-    cy.login(BANK1_MAKER1);
+    cy.saveSession();
```

Comment:
This means that each `it` case continues where it left of in the previous test (instead of having to log in the user etc. each time) so it's more reflective of a real user journey